components/CutoutPreviews.tsx

```diff
@@ -26,67 +28,62 @@ const FileMosaicComponent: React.FC<FileMosaicComponentProps> = ({ singleFile, m
   const hasImages = chunks.length > 0 && chunks.every(chunk => chunk.every(file => file.imageUrl));
 
   return (
-    <>
-      <div className="flex items-center justify-center space-x-4">
-        {singleFile ? (
+    <div className={`flex ${isMobile ? 'flex-col' : 'flex-row'} items-center justify-center space-x-4`}>
+      {singleFile ? (
+        <div className={`${isMobile ? 'mb-5' : 'mr-5'}`}>
           <FileMosaic
             key={singleFile.id}
             {...singleFile}
             info
             preview
             uploadStatus={singleFile.uploadStatus}
-            className="mr-5"
             onSee={() => handleSee(singleFile.imageUrl)}
           />
+        </div>
+      ) : (
+        <CssLoader />
+      )}
+      {isMobile ? null : <FaArrowRight className="text-xl" />}
```

Comment:
**Suggestion:** Consider using a ternary operator instead of a separate null check for `isMobile` in the return statement. This will make the code more concise and easier to read. [enhancement]
```suggestion
    <div className={`flex ${isMobile ? 'flex-col' : 'flex-row'} items-center justify-center space-x-4`}>
      {singleFile ? (
        <div className={`${isMobile ? 'mb-5' : 'mr-5'}`}>
          <FileMosaic
            key={singleFile.id}
            {...singleFile}
            info
            preview
            uploadStatus={singleFile.uploadStatus}
            onSee={() => handleSee(singleFile.imageUrl)}
          />
        </div>
      ) : (
        <CssLoader />
      )}
      {!isMobile && <FaArrowRight className="text-xl" />}
```